### DISCUSSION TITLE
Feature request: lvm size and other sensors

### DISCUSSION THREAD

**Discussion by Gugulator** [dougiteixeira/proxmoxve#113] [Feature request] [answered]
first of all thank you for such great monitoring tool! This is exactly what i was looking for!
I'd like to ask to add some features if it possible. (i know that Proxmox API should provide this information).

1. Sensor for LVM-Thin disk usage. Free and used space in GB and %.
2. Sensor for other disk if it present in system. In specific my case I have /dev/sdb - system, root, swap, data, all lms, and /dev/sda - mounted as directory. So currently this extension does not show information about sda disk.
3. Sensor for successful or unsuccessful last backup for vm/lxc.
4. Sensor with number of working VM as a state, and their names as attributes. Same for LXC.
5. Sensor with current version of Proxmox, and if possible - other sensor giving info about updates.
6. I know that API not provides info about CPU temperature, may be possible add rest or any other kind of sensor, that can use "sensors" command and provide this information.
7. Sensor for storage used in percent, not only free in percent, for all types of storages and swap file. Like you've done it for RAM.

**Comment by dougiteixeira** [comment]
At the moment I don't have the time to invest in these improvements, but I'm happy to merge them if you want to contribute.

**Reply by dougiteixeira** [reply, marked as answer]
The ability to expose storage for the integration was included in #156 (you must go to the integration configuration options to select) and will be available in the next version.

Please open other feature requests for each individual item you want, it makes my job easier. Thanks.
